.github/workflows/zero-downtime-tests.yml

```diff
@@ -12,7 +12,7 @@ on:
       upgradeTo:
         description: 'The helm upgrade command will run with this version (e.g. 0.15.0-dev-PR-7504 or a full address like in the default value )'
         required: true
-        default: 'https://github.com/keptn/helm-charts-dev/raw/1d63e5d3c3f409279a4157d5ad537413abade2e5/packages/keptn-0.16.0-dev.tgz'
+        default: 'https://github.com/keptn/helm-charts-dev/raw/gh-pages/packages/keptn-0.17.0-dev.tgz'
```

Comment:
should we use 0.18.0-dev here as a default instead?

Reply:
I tought to keep it backwar compatible but sure we can